### PR TITLE
Make accessibility fixes

### PR DESCRIPTION
**Pull request by connorpgpmcelroy**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4443

### Change description ###
Fixing accessibility issues

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: src/role-access/containers/allocate-role/choose-duration/choose-duration.component.html [hunk 1]
```diff
@@ -4,104 +4,103 @@ <h1 class="govuk-heading-l">
...
               </div>
             </div>
             <div class="govuk-date-input__item">
               <div class="govuk-form-group">
-                <label class="govuk-label govuk-date-input__label" for="passport-issued-year">
+                <label class="govuk-label govuk-date-input__label" for="passport-issued-year-1">
                 Year
                 </label>
-                <input class="govuk-input govuk-date-input__input govuk-input--width-4" id="passport-issued-year" name="passport-issued-year" type="number" pattern="[0-9]*" formControlName="yearStartDate" 
+                <input id="passport-issued-year-1" name="passport-issued-year-1" type="number" pattern="[0-9]*" formControlName="yearStartDate"
```
Comment (rayliangatos) [inline src/role-access/containers/allocate-role/choose-duration/choose-duration.component.html:59]: ditto

File: src/role-access/containers/allocate-role/choose-duration/choose-duration.component.html [hunk 2]
```diff
@@ -4,104 +4,103 @@ <h1 class="govuk-heading-l">
...
                 </div>
               </div>
               <div class="govuk-date-input__item">
                 <div class="govuk-form-group">
-                  <label class="govuk-label govuk-date-input__label" for="passport-issued-year">
+                  <label class="govuk-label govuk-date-input__label" for="passport-issued-year-2">
                   Year
                   </label>
-                  <input id="passport-issued-year" name="passport-issued-year" type="number" pattern="[0-9]*"  formControlName="yearEndDate" 
+                  <input id="passport-issued-year-2" name="passport-issued-year-2" type="number" pattern="[0-9]*"  formControlName="yearEndDate"
```
Comment (rayliangatos) [inline src/role-access/containers/allocate-role/choose-duration/choose-duration.component.html:103]: ditto

File: src/role-access/containers/allocate-role/choose-duration/choose-duration.component.html [hunk 3]
```diff
@@ -4,104 +4,103 @@ <h1 class="govuk-heading-l">
...
                 </div>
               </div>
               <div class="govuk-date-input__item">
                 <div class="govuk-form-group">
-                  <label class="govuk-label govuk-date-input__label" for="passport-issued-month">
+                  <label class="govuk-label govuk-date-input__label" for="passport-issued-month-2">
                   Month
                   </label>
-                  <input class="govuk-input govuk-date-input__input govuk-input--width-2" id="passport-issued-month" name="passport-issued-month" type="number" pattern="[0-9]*" formControlName="monthEndDate" min="0" max="11"
+                  <input id="passport-issued-month-2" name="passport-issued-month-2" type="number" pattern="[0-9]*" formControlName="monthEndDate" min="0" max="11"
```
Comment (rayliangatos) [inline src/role-access/containers/allocate-role/choose-duration/choose-duration.component.html:94]: ditto

File: src/role-access/containers/allocate-role/choose-duration/choose-duration.component.html [hunk 4]
```diff
@@ -4,104 +4,103 @@ <h1 class="govuk-heading-l">
...
+            <span id="passport-issued-2-hint" class="govuk-hint">
               For example, 12 11 2007
             </span>
             <div class="govuk-error-summary__body">
               <span class="govuk-error-message">
                 {{endDateErrorMessage}}
               </span>
             </div>
-            <div class="govuk-date-input" id="passport-issued">
+            <div class="govuk-date-input" id="passport-issued-2">
```
Comment (rayliangatos) [inline src/role-access/containers/allocate-role/choose-duration/choose-duration.component.html:79]: ditto

File: src/role-access/containers/allocate-role/choose-duration/choose-duration.component.html [hunk 5]
```diff
@@ -4,104 +4,103 @@ <h1 class="govuk-heading-l">
...
           </fieldset>
         </div>
         <div class="govuk-form-group">
-          <fieldset class="govuk-fieldset" aria-describedby="passport-issued-hint" role="group">
+          <fieldset class="govuk-fieldset" aria-describedby="passport-issued-2-hint" role="group">
             <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
               Access ends
             </legend>
-            <span id="passport-issued-hint" class="govuk-hint">
+            <span id="passport-issued-2-hint" class="govuk-hint">
```
Comment (rayliangatos) [inline src/role-access/containers/allocate-role/choose-duration/choose-duration.component.html:71]: ditto

**Review by rayliangatos — APPROVED** [review]
good and speedy fix.

**Review by kuda1992 — APPROVED** [review]
fine

**Pull request merged**